manifests/params.pp

```diff
@@ -8,17 +8,16 @@
   # automatically assign teams for the capstone
   $autoteam  = false
 
-  # list of classes that should be available in the console
-  $classes   = [ 'users', 'apache', 'userprefs' ]
-
   if $::osfamily == 'windows' {
     # Path to the student's working directory
     $workdir = 'c:/puppetcode'
-    $etcpath = 'C:/ProgramData/PuppetLabs/puppet/etc'
+    $confdir = 'C:/ProgramData/PuppetLabs/puppet/etc'
+    $codedir = 'C:/ProgramData/PuppetLabs/puppet/etc'      # TODO
```

Comment:
On Windows the codedir is C:\ProgramData\PuppetLabs\code and the confdir stayed the same as before.